docs/source/intervention_models/crm_mgmt/vivarium_outreach.rst

```diff
@@ -0,0 +1,124 @@
+Vivarium Modeling Strategy - Outreach Intervention 
+--------------------------------------------------
+
+.. todo::
+
+  Add an overview of the Vivarium modeling section.
+
+Decision tree and algorithm for outreach intervention
+
+Visit type: emergency
+
+.. image:: decision_tree_outreach_emergency.svg
+
+Visit type: follow-up
+
+.. image:: decision_tree_outreach_followup.svg
+
+Visit type: screening
+
+.. image:: decision_tree_outreach_screening.svg
+
+Visit type: none
+
+.. image:: decision_tree_outreach_none.svg
+
+Blood pressure ramp: initial prescription
```

Comment:
I recommend that the blood pressure ramp be in a hypertension_treatment.rst file, with sections on the treatment ramp, the initialization strategy, and the effect of the treatment on the SBP risk exposure (for each ramp position)

Reply:
have created separate file; not all details have been added. file is linked to vivarium_{intervention} docs.